.codeclimate.yml-breaks

```diff
@@ -39,3 +39,5 @@ checks:
     channel: "eslint-4"
   markdownlint:
     enabled: true
+  stylelint:
```

Comment:
CodeClimate changed the format of their configuration files. I tried moving over to it, but it seemed to break the build, so I backed off from it. So this file is not actually used currently and this change won't do anything.